src/api/store/community.py

```diff
@@ -1288,6 +1290,26 @@ def update_community_notification_settings(self, context, args) -> Tuple[dict, M
             notification_setting.updated_by = user
             notification_setting.save()
             
+            if activate_on:
+                more_info = notification_setting.more_info or {}
+                task_id = more_info.get("task_id")
+                if task_id:
+                    AsyncResult(task_id).revoke()
+                
+                eta = datetime.strptime(activate_on, "%Y-%m-%d") + timedelta(minutes=0)
+                
+                task_id = automatically_activate_nudge.apply_async(args=[notification_setting.id], eta=eta).id
```

Comment:
Question - queued task survive when a new version of API is deployed? I think so but want to be sure.

Reply:
I am not too sure about it but I think they do survive